From b04fcc43f602d865f9f5d4fdf90de47943e0027b Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentfredric@gmail.com>
Date: Fri, 9 Jun 2017 10:19:57 +1200
Subject: [PATCH] Fix spec.PL for Perl 5.26 '.' removal from @INC

Perl 5.26 removes '.' from @INC, and this breaks the semantics
of <<do $file>> where $file was assumed to be loaded from '.',
but not explicitly stated, as 'do' is implemented using the same code
as "require", that is, assuming @INC traversal.

Inserting a leading "./" forces do to use a special-cased path
where @INC is not consulted.

Fixes RT#120697

Bug: https://rt.cpan.org/Ticket/Display.html?id=120697
---
 perl-Sys-Virt.spec.PL | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/perl-Sys-Virt.spec.PL b/perl-Sys-Virt.spec.PL
index aa3f6fc..d2d687c 100644
--- a/perl-Sys-Virt.spec.PL
+++ b/perl-Sys-Virt.spec.PL
@@ -6,10 +6,10 @@ use strict;
 
 die unless (scalar @ARGV == 1);
 
-unless (do 'lib/Sys/Virt.pm')
+unless (do './lib/Sys/Virt.pm')
   {
     if ($@) { die $@ };
-    die "lib/Sys/Virt.pm: $!";
+    die "./lib/Sys/Virt.pm: $!";
   }
 
 local $/ = undef;
-- 
2.13.1

